topics/assembly/tutorials/ecoli_comparison/workflows/empty_workflow.ga

```diff
@@ -0,0 +1,8 @@
+{
+    "a_galaxy_workflow": "true", 
```

Comment:
Can you add the workflow here?

Reply:
there is no workflow here

Reply:
You could export your history and use this workflow. The workflow you could use to create the tools.yaml file. The overall idea behind workflows is that we use them for automatic testing of tutorials but also give trainees a good overview of what was/will be done.